vue/sbc-common-components/src/components/SbcFeeSummary.vue

```diff
@@ -16,7 +16,7 @@
         <li class="container fee-list__item"
           :key="lineItem.filingType"
           >
-          <div class="fee-list__item-name">{{lineItem.filingType}}</div>
+          <div class="fee-list__item-name">{{filingLabel ? filingLabel : lineItem.filingType}}</div>
```

Comment:
FYI, the other way to do this is `filingLabel || lineItem.filingType`.

But this is fine.

Reply:
This implementation sort of falls down is there are multiple line items (they will all use the override label).

Probably OK for now but may need to be fixed in the future if there are multiple fees and we need to override a fee label.